### PR TITLE
[Typo] update some out-dated links from pytorch ``clip_grad_value_``

### LINKED ISSUE #8532: [typo] link of ``clip_grad_norm``
## 📚 Documentation

For typos and doc fixes, please go ahead and:

1. Create an issue.
2. Fix the typo.
3. Submit a PR.

Thanks!

links in [this lines](https://github.com/PyTorchLightning/pytorch-lightning/blob/dd340a659818def94468f11aa4fed492e27f3e92/pytorch_lightning/utilities/enums.py#L104) about ``clip_grad_norm`` and ``clip_grad_value``.
These two should be [value](https://pytorch.org/docs/stable/nn.html#torch.nn.utils.clip_grad_value_) and [norm](https://pytorch.org/docs/stable/nn.html#torch.nn.utils.clip_grad_norm_).

### PR DESCRIPTION
**Pull request by hihunjin**
## What does this PR do?

Fixes #8532

## Before submitting
- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)
- [ ] Did you list all the **breaking changes** introduced by this pull request?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 😃

### CODE REVIEW

**Comment by hihunjin** [comment]
If the link like [this](https://pytorch.org/docs/stable/generated/torch.nn.utils.clip_grad_norm_.html#torch.nn.utils.clip_grad_norm_) looks better than [this link](https://pytorch.org/docs/stable/nn.html#torch.nn.utils.clip_grad_norm_), please tell me. I'll fix them.

File: pytorch_lightning/utilities/enums.py [hunk 1]
```diff
@@ -106,8 +106,8 @@ class GradClipAlgorithmType(LightningEnum):
     VALUE tpye means "clipping gradients by value". This will clip the gradient value for each parameter.
```
Comment (carmocca) [inline pytorch_lightning/utilities/enums.py:106]:
```suggestion
    VALUE type means "clipping gradients by value". This will clip the gradient value for each parameter.
```

**Review by carmocca — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
